compiler/src/iree/compiler/Dialect/Flow/Transforms/FormDispatchRegions.cpp

```diff
@@ -502,6 +502,18 @@ static bool canUseInOperandAsInitOperand(OpOperand *inOperand,
   return true;
 }
 
+/// All operations in a dispatch should be vectorized, which isnt the case today
+/// This is an explicit list of operations that arent vectorized for now
+/// requiring special handling for now in dispatch region formation to avoid
+/// large stack allocations.
+static bool isVectorizedAlways(Operation *producer) {
+  if (auto convOp = dyn_cast<linalg::Conv2DNhwcHwcfOp>(producer)) {
```

Comment:
What about other kinds of conv ops? You could use `linalg::inferConvolutionDims` to get the strides and dilations for anything "convolution-like"

Reply:
this is kind of a black list that we grow if needed. Eventually this should just return true always.